docs/recipes/auth/login_form.md

```diff
@@ -0,0 +1,479 @@
+			rx.center(
+				rx.text("New here?", size="3"),
+				rx.link("Sign up", href="#", size="3"),
+				opacity="0.8",
+				spacing="2",
+				direction="row",
+				width="100%"
+			),
+			spacing="6",
+			width="100%"
+		),
+		size="4",
+		max_width="28em",
+		width="100%"
+	)
+```
+
+## Default login card with icons
+
+```python exec
+def login_default_icons() -> rx.Component:
+	return rx.card(
+		rx.vstack(
+			rx.center(
+				rx.image(src="/favicon.ico", width="2.5em", height="auto", border_radius="50%"),
+				rx.heading("Sign in to your account", size="6", as_="h2", text_align="center", width="100%"),
+				direction="column",
+				spacing="5",
+				width="100%"
+			),
+			rx.vstack(
+				rx.text("Email address", size="3", weight="medium", text_align="left", width="100%"),
+				rx.input(rx.input.slot(rx.icon("user")), placeholder="user@reflex.dev", type="email", size="3", width="100%"),
+				spacing="2",
+				width="100%"
+			),
+			rx.vstack(
+				rx.hstack(
+					rx.text("Password", size="3", weight="medium"),
+					rx.link("Forgot password?", href="#", size="3"),
+					justify="between",
+					width="100%"
+				),
+				rx.input(rx.input.slot(rx.icon("lock")), placeholder="Enter your password", type="password", size="3", width="100%"),
+				spacing="2",
+				width="100%"
+			),
+			rx.button("Sign in", size="3", width="100%"),
+			rx.center(
+				rx.text("New here?", size="3"),
+				rx.link("Sign up", href="#", size="3"),
+				opacity="0.8",
+				spacing="2",
+				direction="row",
+				width="100%"
+			),
+			spacing="6",
+			width="100%"
+		),
+		max_width="28em",
+		size="4",
+		width="100%"
+	)
+```
+
+```python demo box
+login_default_icons()
+```
+
+```python
+def login_default_icons() -> rx.Component:
+	return rx.card(
+		rx.vstack(
+			rx.center(
+				rx.image(src="/favicon.ico", width="2.5em", height="auto", border_radius="50%"),
+				rx.heading("Sign in to your account", size="6", as_="h2"),
+				direction="column",
+				spacing="5",
+				width="100%"
+			),
+			rx.vstack(
+				rx.text("Email address", size="3", weight="medium"),
+				rx.input(rx.input.slot(rx.icon("user")), placeholder="user@reflex.dev", type="email", size="3", width="100%"),
+				spacing="2",
+				width="100%"
+			),
+			rx.vstack(
+				rx.hstack(
+					rx.text("Password", size="3", weight="medium"),
+					rx.link("Forgot password?", href="#", size="3"),
+					justify="between",
+					width="100%"
+				),
+				rx.input(rx.input.slot(rx.icon("lock")), placeholder="Enter your password", type="password", size="3", width="100%"),
+				spacing="2",
+				width="100%"
+			),
+			rx.button("Sign in", size="3", width="100%"),
+			rx.center(
+				rx.text("New here?", size="3"),
+				rx.link("Sign up", href="#", size="3"),
+				opacity="0.8",
+				spacing="2",
+				direction="row",
+				width="100%"
+			),
+			spacing="6",
+			width="100%"
+		),
+		size="4",
+		max_width="28em",
+		width="100%"
+	)
+```
+
+## Login card with third-party auth provider
```

Comment:
Long titles sometime spills, also consider simplify the titles. We don't need navbar in name of every variants. 
![Screenshot 2024-05-23 at 12 17 44](https://github.com/reflex-dev/reflex-web/assets/159659100/09a4cdc2-4d88-4d95-950f-9ff78a86cfd2)
![Screenshot 2024-05-23 at 12 18 19](https://github.com/reflex-dev/reflex-web/assets/159659100/77a1bccb-43ae-4a8c-9530-f12c96d46b5a)